identity/client/src/pages/groups/modals/AddModal.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+import { FC, useEffect, useState } from "react";
+import { FormModal, UseModalProps } from "src/components/modal";
+import useTranslate from "src/utility/localization";
+import { useApiCall } from "src/utility/api/hooks";
+import TextField from "src/components/form/TextField";
+import { createGroup } from "src/utility/api/groups";
+
+const AddModal: FC<UseModalProps> = ({ open, onClose, onSuccess }) => {
+  const { t } = useTranslate();
+
+  const [callAddGroup, { loading, namedErrors }, reset] =
+    useApiCall(createGroup);
+
+  const [name, setName] = useState("");
+
+  const handleSubmit = async () => {
+    const { success } = await callAddGroup({
+      name: name.trim(),
+    });
+
+    if (success) {
+      onSuccess();
+    }
+  };
+
+  useEffect(() => {
+    if (open) {
+      setName("");
+      reset();
+    }
+  }, [open]);
```

Comment:
Resetting the modal state is not needed (any more) because the hook `useModal` takes care of onmounting the modal if it is closed, which will also reset the internal state. https://github.com/camunda/camunda/pull/19368/files#diff-5ca07a5b64db73bfe3e5a0301440bcb425bc65a0d097885524e439764b0b96eaR36

```suggestion
```